### ISSUE TITLE
Line: 182 Error: Invalid argument.

### ISSUE THREAD

**Issue by mraible** [louisremi/background-size-polyfill#11]
I'm using [Bootstrap's Tabs feature](http://twitter.github.io/bootstrap/javascript.html#tabs) to show different images on each tab. For some reason, imgRatio is NaN on line 177, so the following line fails with "Invalid argument".

```
img.style.left = delta + "px";
```

Changing this line to the following seems to fix the problem.

```
if (!isNaN(delta)) {
    img.style.left = delta + "px";
}
```

**Comment by frebro** [comment]
I'm seeing the same error on line 182 and line 220 when using FlexSlider. The reason seems to be that the image  does not have a dimension yet. I can confirm that 2e487298606dc125228e339c73fc82ea88a7b590 fixes this.

**Comment by jefferyto** [comment]
This should be fixed in 0.2.0. Please reopen (or file a new ticket) if the issue reappears - thanks :-)
